.github/workflows/tests.yml

```diff
@@ -2,63 +2,60 @@ name: Tests
 on:
   push:
     branches:
-    - main
-    - next
-    - v1
+      - main
+      - next
+      - v1
   pull_request:
     branches:
-    - main
-    - next
-    - v1
+      - main
+      - next
+      - v1
 jobs:
-
   unit:
     name: API Tests
     runs-on: ubuntu-latest
     steps:
-    - name: Checkout repo
-      uses: actions/checkout@v3
-    - name: Setup node
-      uses: actions/setup-node@v3
-      with:
-        node-version: 20
-    - uses: pnpm/action-setup@v2
-      with:
-        version: 8
-    - name: Download deps
-      run: pnpm i
-    - name: Build packages
-      run: pnpm run build
-    - name: Run tests
-      run: pnpm exec vitest
-    - name: Typecheck
-      run: pnpm -r run typecheck
+      - name: Checkout repo
+        uses: actions/checkout@v3
+      - name: Setup node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 20
+      - uses: pnpm/action-setup@v2
+        with:
+          version: 8
+      - name: Download deps
+        run: pnpm i
+      - name: Build packages
+        run: pnpm run build
+      - name: Run tests
+        run: pnpm exec vitest
+      - name: Typecheck
+        run: pnpm -r run typecheck
 
   e2e:
     name: E2E Tests
     runs-on: ubuntu-latest
     steps:
-    - name: Checkout repo
-      uses: actions/checkout@v3
-    - name: Setup node
-      uses: actions/setup-node@v3
-      with:
-        node-version: 20
-    - uses: pnpm/action-setup@v2
-      with:
-        version: 8
-    - name: Download deps
-      run: pnpm i
-    - name: Install Playwright
-      run: pnpm exec playwright install --with-deps
-    - name: Build packages
-      run: pnpm run build
```

Comment:
this is the only thing that changed, the rest was format issues with prettier I believe.